mongostat/mongostat.go

```diff
@@ -243,8 +243,12 @@ func NewNodeMonitor(opts options.ToolOptions, fullHost string) (*NodeMonitor, er
 	}
 	uriCopy.ConnectionString = newCS
 	optsCopy.URI = &uriCopy
-	optsCopy.Direct = true
-	optsCopy.ConnString.Hosts = []string{fullHost}
```

Comment:
Since we are not replacing the `Hosts` in ConnString with single node `fullHost`, the above code 
```	newCS, err := rewriteURI(uriCopy.ConnectionString, fullHost)
	if err != nil {
		return nil, err
	}
```
doesn't make sense in a replica-set connection anymore. I guess it didn't affect the tests, but it might have some side impact. 

I suggest we put the `rewriteURI` in the below if condition as well.

Reply:
Yeah, that's a good point!